models/dataset.go

```diff
@@ -212,6 +213,8 @@ func CreateVersion(reader io.Reader) (*Version, error) {
 		return nil, errors.New("Failed to read message body")
 	}
 
+	log.Info("version", log.Data{"bytes": string(bytes)})
```

Comment:
This log message could probably be more descriptive - and we need to check if we're happy to log out the whole version doc or if we'd consider any of that data sensitive

Reply:
Sorry, I was using this for debugging; I should have removed